pangocffi/__init__.py

```diff
@@ -1,5 +1,16 @@
 import ctypes.util
-from ._generated.ffi import ffi
+from .ffi_build import ffi
+import logging
+
+fmtstr = "%(asctime)s: (%(filename)s): %(levelname)s: " \
+         "%(funcName)s Line: %(lineno)d - %(message)s"
+logging.basicConfig(
```

Comment:
I would suggest using `getLogger` method here.
https://docs.python.org/3/howto/logging-cookbook.html#using-logging-in-multiple-modules

Reply:
Also, does it needed to be written in a file?

Reply:
I added logging to try to understand how python imports modules... I don't intend to keep it 😄

I'm hoping that multiple calls of `import pangocffi` from external modules only calls `__init__.py` once, which means the FFI can be reused within the same runtime session. If it gets called twice, then there's a problem because it will be creating two FFI instances, which will break things.

From some brief testing, this appears to be the case, and the message `__init__.py called from pangocffi` gets logged once, even with multiple calls to `import pangocffi`!

I hope to make a release at the latest by the end of this week with this change... I need to do more testing, hopefully with https://github.com/manimcommunity/manim (Thanks for pointing me to it btw!)

Reply:
I don't really know more about this. But for some reasons, it didn't work **Out of the Box** on Windows and we got a lot of Issue because of that on Manim. 
> I hope to make a release at the latest by the end of this week with this change... I need to do more testing, hopefully with https://github.com/manimcommunity/manim (Thanks for pointing me to it btw!)

I was the one to implement that over there. If you pick the release version you would need to [`PangoText`](https://docs.manim.community/en/stable/reference/manim.mobject.svg.text_mobject.PangoText.html#manim.mobject.svg.text_mobject.PangoText) use the example script over there and also you could over to out [discord server](https://discord.gg/mMRrZQW) if you need help. 

Also, I would soon possibly make it optional because of the issue it cause 
see [[1]](https://docs.manim.community/en/latest/installation/troubleshooting.html#modulenotfounderror-no-module-named-generated-ffi) and [[2]](https://docs.manim.community/en/latest/installation/win.html#pango-installation). 
Hopefully, the first one will be fixed soon.